3-extensions/registry/dubbo-samples-nacos/dubbo-samples-nacos-registry/dubbo-samples-nacos-registry-provider/src/main/resources/application.yml

```diff
@@ -0,0 +1,25 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+dubbo:
+   application:
+        name: dubbo-samples-nacos-registry-consumer
+        qos-port: 20993
+   protocol:
+        name: dubbo
+        port: 20883
```

Comment:
Make sure this port is the same as the value here  https://github.com/apache/dubbo-samples/pull/1086/files#diff-162f44ea31c4096aa09a7da9b61f87efeded56aefa1811e8d9a4a1be91bf05f8R40

Please use `20880`